apps/cli/src/index.ts

```diff
@@ -136,6 +136,22 @@ cli.action(async () => {
             cli.outputHelp();
             continue;
         }
+        
+        // Show help message if the user asks for it
+        if (command === 'toggle-switch-network') {
+            if(config.defaultNetworkName === "arbitrum"){
```

Comment:
Should this be arbitrumSepolia instead?

Reply:
I don't think it should.
If the current config's defaultNetworkName equals "arbitrum" we are currently using mainnet (default) so we switch to testnet, if not we switch back to mainnet.